dm/worker/log.go

```diff
@@ -340,7 +351,8 @@ func (logger *Logger) doGC(db *leveldb.DB, id int64) {
 	iter.Release()
 	err := iter.Error()
 	if err != nil {
-		log.Errorf("[task log gc] query logs from meta error %v", err)
+		log.Errorf("[task log gc] query logs from meta error %v in range [%s(% X), %s(% X))",
```

Comment:
what is the use of this log?

Reply:
just log the _range_ if _iter_ error for debugging.